# Generated by Django 3.2.15 on 2023-02-10 18:19

from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        ('contenttypes', '0002_remove_content_type_name'),
        ('data_files', '0012_datafile_s3_versioning_id'),
        ('search_indexes', '003_auto_20230127_1553'),
    ]

    operations = [
        migrations.CreateModel(
            name='ParserError',
            fields=[
                ('id', models.AutoField(primary_key=True, serialize=False)),
                ('row_number', models.IntegerField()),
                ('column_number', models.IntegerField()),
                ('item_number', models.IntegerField()),
                ('field_name', models.TextField(max_length=128)),
                ('category', models.IntegerField(default=1)),
                ('error_message', models.TextField(max_length=512, null=True)),
                ('error_type', models.TextField(max_length=128)),
                ('object_id', models.PositiveIntegerField()),
                ('created_at', models.DateTimeField(auto_now_add=True)),
                ('content_type', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='contenttypes.contenttype')),
                ('file', models.ForeignKey(null=True, on_delete=django.db.models.deletion.CASCADE, related_name='parser_errors', to='data_files.datafile')),
            ],
            options={
                'db_table': 'parser_error',
            },
        ),
    ]
